apps/web/src/services/carelocations.tsx

```diff
@@ -0,0 +1,5 @@
+import axios from 'axios';
+
+export const getCareLocations = async () => {
+  return (await axios.get('/carelocations')).data;
```

Comment:
Ahhh, I'm a little skeptical about this approach. I'll make a change to this PR, let me know if that looks good.

Reply:
Please do let me know, thank you!